flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -32,12 +35,14 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
-		9705A1C41CF9048500538489 /* Embed Frameworks */ = {
+		19C134552876922700263A48 /* Embed Frameworks */ = {
 			isa = PBXCopyFilesBuildPhase;
 			buildActionMask = 2147483647;
 			dstPath = "";
 			dstSubfolderSpec = 10;
 			files = (
+				19C134542876922700263A48 /* backend_bridge_fw.xcframework in Embed Frameworks */,
+				19C1345828769D8300263A48 /* tflite_backend_fw.xcframework in Embed Frameworks */,
```

Comment:
One issue I wasn't able to solve is that XCode tries to bundle all listed frameworks. If we were to disable TFLite backend we would get a compile error about missing framework.
We will need to create an empty dummy backend to include it instead of the real one if we want to be able to disable it (without using xcode UI).